superset/connectors/sqla/models.py

```diff
@@ -113,7 +113,7 @@ def get_time_filter(self, start_dttm, end_dttm):
     def get_timestamp_expression(self, time_grain):
         """Getting the time component of the query"""
         expr = self.expression or self.column_name
-        if not self.expression and not time_grain:
+        if not self.expression and (not time_grain or time_grain == u'Time Column'):
```

Comment:
`u'foo'` isn't python3 compatible I think. I'm surprised that label-looking `Time Column` is a possible value here, somehow it should be `time_column` or `None` instead.

Reply:
Yea, it's u'Time Column' not None or something else

Reply:
I looked up the python 3 docs, the u'foo' is valid in python 3